blobfuse/utilities.cpp

```diff
@@ -281,7 +281,7 @@ int is_directory_empty(std::string container, std::string delimiter, std::string
             success = false;
             failcount++; //TODO: use to set errno.
         }
-    } while ((continuation.size() > 0) && !success && (failcount < 20));
+    } while (continuation.size() > 0 || (!success && failcount < 20));
```

Comment:
This will loop forever if there are failures on a request with a continuation. Should be (cont.size>0 && success) || (!success && failcount<20). This should be refactored to avoid such a complex condition, but that can be done later.

Reply:
ok, will do